extensions/functions_datetime.yaml

```diff
@@ -193,3 +203,66 @@ scalar_functions:
           - value: interval_year
           - value: interval_year
         return: boolean
+  - name: "assume_timezone"
+    description: Convert local time to UTC using given local time's timezone.
+    impls:
+      - args:
+          - options: [ TIMEZONE ]
+            name: Timezone of provided local time.
+            required: true
+          - value: timestamp
+        return: timestamp_tz
+  - name: "strptime"
+    description: Parse string into timestamp/date/time using provided format.
```

Comment:
We need to declare the format rules this function uses.

Reply:
Would that go into description here? I would start by adopting https://man7.org/linux/man-pages/man3/strptime.3.html